.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "automatic"
+}
```

Comment:
Typically we haven't been adding IDE related files (in this case VSCode specific) to Git. Should we remove this file from Source control?

Reply:
meh. is harmless.  I can delete it if you really don't want it.

Reply:
i've removed it